group.go

```diff
@@ -134,6 +136,176 @@ func (client *Client) ListGroups() ([]Group, error) {
 	return q.Account.Groups.Query(client, &q, v)
 }
 
+func (g *Group) DescendantTeams(client *Client) ([]Team, error) {
+	var q struct {
+		Account struct {
+			Group struct {
+				DescendantTeams struct {
+					Nodes    []Team
+					PageInfo PageInfo
+				} `graphql:"descendantTeams(after: $after, first: $first)"`
+			} `graphql:"group(id: $group)"`
+		}
+	}
+	if g.Id == nil {
+		return nil, fmt.Errorf("failed to request 'DescendantTeams' for group because there is not a valid id '%s'", g.Id)
+	}
+	v := PayloadVariables{
+		"group": g.Id,
+		"first": client.pageSize,
+		"after": g.DescendantTeamsConnection.PageInfo.Start,
+	}
+	output := []Team{}
+	if err := client.Query(&q, v); err != nil {
+		return nil, err
+	}
+	output = append(output, q.Account.Group.DescendantTeams.Nodes...)
+	for q.Account.Group.DescendantTeams.PageInfo.HasNextPage {
+		v["after"] = q.Account.Group.DescendantTeams.PageInfo.End
+		if err := client.Query(&q, v); err != nil {
+			return nil, err
+		}
+		output = append(output, q.Account.Group.DescendantTeams.Nodes...)
```

Comment:
Do teams have sub resources that have pagination - i think they do.  We might need to convert all these to only return `TeamID` types to not mess with the recursive pagination.

Reply:
Yeah, I think the members field.

Reply:
Then ya another case for TeamId